### ISSUE TITLE
In multi-root workspace Terminal Split doesn't work.

### ISSUE THREAD

**Issue by oswee** [Microsoft/vscode#54061]
We have written the needed data into your clipboard because it was too large to send. Please paste.

**Comment by Tyriar** [comment]
What doesn't work about it? Note that if you wrote out a description in the issue reporter you were meant to paste it into the GitHub issue (/cc @RMacfarlane)

**Comment by oswee** [comment]
Yeah... that description pasting thing is strange. Should work with single click and without any additional actions.

I made multi-root workspace with 3 repositories. Made editor grid of 3 columns. Open Terminal. Try to split it. - NOT WORKING.
Go back to regular (no workspace) mode - Terminal splitting works as expected.
What to explain more?

**Comment by Tyriar** [comment]
@oswee did you have the terminal focused when you tried to split it? Also were you using the command/command palette or the button on the panel to split?
